src/components/map/MapService.js

```diff
@@ -968,87 +974,14 @@ goog.require('ga_urlutils_service');
                 }
               });
 
-              // Test extent define in config for tiled WMS an WMTS
-              response.data['ch.bfe.sachplan-geologie-tiefenlager'].extent =
-                  [635432, 182850, 708432, 291850];
-              response.data['ch.vbs.patrouilledesglaciers-z_rennen'].extent =
-                  [582382, 92500, 625032, 105700];
```

Comment:
We decided to ignore this feature for now...but we should get back to it eventually.